reduces the price for most livestock crates

Donglesplonge opened the pull request:
## About The Pull Request

this ones always been itching at the back of my head, mainly because i'm greedy, and you should be too, animals in cargo are hardcore expensive, 2000 credits for a cat? 800 credits per single chicken? 1200 per pig? obscene! this PR lowers the costs across the board with some exceptions, which i'll note here: 

crab rockets remain the same price as they were, you get 50 crabs PER rocket, thats ALOT of crabs per cubic inch, what would you do with so many? i don't know, but its why they're remaining expensive, at 1600 per rocket.


butterfly crates also retain their original price, as you also get 50 butterflies per crate purchased, at a cost of 1000 credits for a crate of 50, i think this is fair.


monkey cubes, truth be told this one i'm kinda scratching my head on, they're remaining the same price right now due to how accessible they are from other avenues (xenobiology, botany both public and private), but i'm willing to change their price to accommodate if they seem to be needing a price reduction, but overall i think that theres alot of ways to accrue them so them remaining particularly expensive isn't necessarily a bad thing. 


one part of this PR that bothered me was that i couldn't figure out how to display gnomes on either the cargo console or the express console, but they also have been price slashed, down from 4000 credits per 3 gnomes to 3000 credits per 3 gnomes, overall a 25% decrease in cost, think of all the mystical forest friends you can have for that price!

EDIT: i forgot about duct spiders, whoopsy me, they aren't in the livestock crates tab and are considered imports instead, so it slipped my mind completely, they are down from 1200 credits to 800, still moderately expensive, but still a discount nonetheless.


heres two pictures displaying the before-the-price decrease values, and the after-the-price- decrease values: 

BEFORE 
![Screenshot 2024-06-19 002027](https://github.com/tgstation/tgstation/assets/120208006/d0d5925d-d489-41e0-8345-e22fbd5cdd09)

  AFTER
![Screenshot 2024-06-19 005017](https://github.com/tgstation/tgstation/assets/120208006/824c28e1-2103-4cf1-ab77-077a845318e1)

i'll gladly edit or change any prices that seem way too low, and provide what i believe are reasonable answers for all of them, and i'd gladly love feedback from anyone on whether or not they think somethings too low, or still too high.


## Why It's Good For The Game

buying animals from cargo is generally a very difficult think to pull off for mass gimmicks due to the difficulty of accruing wealth crew-side in the jobs that would more than likely attempt to participate in such gimmicks (assistants, for one) so i think cutting their prices will not only help people actually be able to realize their gimmicks in round, but also drum up more cash for cargo, as you can now reasonably buy multiple animals without immediately going bankrupt and having to farm exorbitant amounts of money that you probably won't be able to make within the scope of an average round without some truly exotic methods, i think this will improve the cashflow for the livestocks tab, while also healthily promoting peoples funny little critter gimmicks.

## Changelog

:cl:
balance: most livestock crates, with some exception, have been made cheaper to facilitate healthier mental states in the crew, go build a farm!
/:cl:

Donglesplonge committed:
cargo animal price changes

changes the value of cargo animals (for the most part) to be cheaper across the board, with one notable exception

Donglesplonge committed:
forgot about duct spiders

completely forgot about one of my favorite cargo animal orders, shame on me

Donglesplonge commented:
as i said in the PR: if you think prices are way too low, even remotely so, please do tell me, despite me wanting them to be cheaper, ruining cargos cashflow from it would be horrendous, and i don't want to step on their toes over this kinda stuff, as i'm not a cargo main, nor have i really ever played cargo, i don't exactly know if the prices are well-adjusted for it, i just really like buying lots of animals so i'm angling from the crew side.

DaCoolBoss commented:
Ponies and exotic corgies should be more expensive than most other animals, they're status symbols. I say leave their prices at the old level.

One of the problems with very cheap crates are that you get a 200 credit refund from the crate and another 80 credits or so from a correctly stamped manifest, so those 400 credit crates are only 120 credits after shipping the crate back. If the cheap cargo station trait rolls (20% off all sticker prices) this becomes only 40 credits. Might be worth bumping some of those 400 credit prices to 500 (almost double the effective cost after refunding crates) if spamming is an issue.

Donglesplonge commented:
> Ponies and exotic corgies should be more expensive than most other animals, they're status symbols. I say leave their prices at the old level.

exotic corgis i could bump back up, they're separate from the rest of the animals on their own tier of pricing, the reason ponies are as cheap as some others is because ponies were on the same tier of price as some other animals, so the way the crates work in code is like this 

cargos base crate price, which is 200, multiplied by a set number 

in this instance, ponies were  200 x 6, which is 1200, along with stuff like the pigs, cows, and snakes, i'd generally want them to all be on the same bracket, price-wise, so there isn't just a bunch of random crates with all differing values for no specific reason,  its a personal reason but nonetheless i would be willing to bump everything in the price bracket up if nobody cares, but i'd still wish for the bracket to remain generally not as expensive

however, i will probably bump exotic corgis specifically up a little more, i think thats reasonable, they are meant to be the most "exotic" of the animals you can purchase and thus should probably incur a heavier price, also they're alone in their price bracket, so it doesn't fuck with my head trying to rationalize the rest of it

Donglesplonge commented:
> > Ponies and exotic corgies should be more expensive than most other animals, they're status symbols. I say leave their prices at the old level.
> 
> exotic corgis i could bump back up, they're separate from the rest of the animals on their own tier of pricing, the reason ponies are as cheap as some others is because ponies were on the same tier of price as some other animals, so the way the crates work in code is like this
> 
> cargos base crate price, which is 200, multiplied by a set number
> 
> in this instance, ponies were 200 x 6, which is 1200, along with stuff like the pigs, cows, and snakes, i'd generally want them to all be on the same bracket, price-wise, so there isn't just a bunch of random crates with all differing values for no specific reason, its a personal reason but nonetheless i would be willing to bump everything in the price bracket up if nobody cares, but i'd still wish for the bracket to remain generally not as expensive
> 
> however, i will probably bump exotic corgis specifically up a little more, i think thats reasonable, they are meant to be the most "exotic" of the animals you can purchase and thus should probably incur a heavier price, also they're alone in their price bracket, so it doesn't fuck with my head trying to rationalize the rest of it

forgot to mention the part about the 400 credit stuff, i could look into putting them at 500 per, but the little voice in the back of my head is telling me its stretching the value, as for most of the crates in the 400 bracket, you only get 1 animal per, i might look into seeing if it'd be an alright move to make it so you get 2 per instead and then push the price upwards, so that way its not a slap in the face for either party trying to get what they'd want from the exchange

Donglesplonge commented:
i swear to god i keep quote replying myself on accident and making messages even dumber this isn't on purpose i swear i'm actually crying right now

Donglesplonge committed:
exotic corgi price increase

increases the price of the exotic corgi to being more inline with its exotic nature

Donglesplonge commented:
exotic corgis are now 1400 credits per crate, to be in line with their exotic nature as per request, being the most expensive normal livestock crate to purchase by 400 credits.

san7890 commented:
your changelog was borked, i fixed it

san7890 commented:
@ArcaneMusic this seems okay to me, what are your thoughts?

Donglesplonge commented:
> your changelog was borked, i fixed it

thank you i hadn't noticed, i appreciate it

Loafin34 commented:
make ponies more expensive. shit's as fast as meth when you have high mood

Donglesplonge committed:
pony price increase

ponies do provide a pretty nice speed boost, so its not unreasonable that they be a little more expensive

Donglesplonge commented:
> make ponies more expensive. shit's as fast as meth when you have high mood


pushed up to 1000 credits, which i think is a more fair price, truth be told i wasn't thinking much on them but now that i do i agree, if its still too low we'll have to see, i can always just PR it to be higher later if its too much

DrAmazing343 commented:
> exotic corgis are now 1400 credits per crate, to be in line with their exotic nature as per request, being the most expensive normal livestock crate to purchase by 400 credits.

absolutely fucking based PR. speedmerge instantly AFTER making exotic corgis 5,000 per crate. I want to be the sole and exclusive exotic corgi owner in this daringly sparse market.

On a more serious note, does anyone ever buy them? I think leaving them at 1400 is enough to be a little funny while also maintaining the reasonable idea of the mob, but... who ever actually buys them?

Donglesplonge commented:
> > exotic corgis are now 1400 credits per crate, to be in line with their exotic nature as per request, being the most expensive normal livestock crate to purchase by 400 credits.
> 
> absolutely fucking based PR. speedmerge instantly AFTER making exotic corgis 5,000 per crate. I want to be the sole and exclusive exotic corgi owner in this daringly sparse market.
> 
> On a more serious note, does anyone ever buy them? I think leaving them at 1400 is enough to be a little funny while also maintaining the reasonable idea of the mob, but... who ever actually buys them?

Exotic corgis arent particularly unique or cool looking, pretty much just a corgi but colored differently, so yeah id reckon they arent picked all that much for that exclusively, but i havent seen them ordered enough or even ordered them myself to know if what im saying is even true or not, someone couldve ghostmerged them firing deathbolts every time they barked and nobody would know

ExcessiveUseOfCobblestone commented:
Was there like a system you used to determine prices or did you just line by line checked stuff out

Donglesplonge commented:
> Was there like a system you used to determine prices or did you just line by line checked stuff out

went line by line, there truthfully wasn't enough animals or majorly unique pricings for animals so it wasn't all that difficult, i kept all of them to at most being a half off slash, as to not completely destroy all the values of the higher tiers while also making the lower tiers basically free, but yeah it was me seeing "oh pigs and sheep are the same price, so they'll keep the same price after the numbers changed but it will still be lower than their original price"

ArcaneMusic commented:
> @ArcaneMusic this seems okay to me, what are your thoughts?

I think for the most part there's nothing here that's too concerning, the pony change seems fine. I personally would prefer if we kept exotic corgies at the same price as it's more of a flex symbol than a mechanical benefit so leaving it at a relatively high price seems fine, perhaps meeting halfway at 1400 cr?

Donglesplonge commented:
> I think for the most part there's nothing here that's too concerning, the pony change seems fine. I personally would prefer if we kept exotic corgies at the same price as it's more of a flex symbol than a mechanical benefit so leaving it at a relatively high price seems fine, perhaps meeting halfway at 1400 cr?


I bumped them up to 1400 baseline earlier in the PR, because i did agree that theyre supposed to be status symbols, so theyre the most expensive normal crate by 400 creds

ArcaneMusic reviewed (approved):
We ball
